comment_scan/commander.rb

```diff
@@ -0,0 +1,441 @@
+require 'time'
+require './db'
+
+class Commander
+    attr_reader :chatter, :seclient, :scanner, :BOT_NAMES, :start_time
+
+    def initialize(chatter, seclient, scanner, bot_names)
+        @chatter = chatter
+        @seclient = seclient
+        @scanner = scanner
+        @BOT_NAMES = bot_names
+
+        @basic_commands = Hash.new()
+        @HQ_commands = Hash.new()
+
+        @basic_commands["!!/whoami"] = method(:whoami)
+        @basic_commands["!!/alive"] = method(:alive)
+        @basic_commands["!!/on"] = method(:on)
+        @basic_commands["!!/off"] = method(:off)
+        @basic_commands["!!/mode"] = method(:mode)
+        @basic_commands["!!/help"] = method(:help)
+        @basic_commands["!!/notify"] = method(:notify)
+        @basic_commands["!!/reports"] = method(:reports)
```

Comment:
This is really neat, I love the way you did this.

Reply:
Thank you!!